internal/mock_api/mock_server/server.go

```diff
@@ -101,6 +101,7 @@ func loggerMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log.Printf("%v %v", r.Method, r.URL.Path)
 		w.Header().Set("Content-Type", "application/json")
+		w.Header().Set("access-control-allow-origin", "*")
```

Comment:
Very minor, but can you make this "Access-Control-Allow-Origin" instead? Just want to match docs externally and use a consistent formatting for headers.

Reply:
Done!